src/renderer/components/product/tabs.vue

```
@@ -1,8 +1,8 @@
 <template>
   <Tabs :model-value="currentTab" @update:model-value="handleUpdateTab" class="max-h-[90vh] w-full">
-    <TabsList class="grid w-full grid-cols-3">
+    <TabsList class="grid w-full grid-cols-4">
       <TabsTrigger :value="MULMO_VIEWER_TABS.MOVIE">{{ t("project.productTabs.tabs.movie") }}</TabsTrigger>
-      <!-- TabsTrigger :value="MULMO_VIEWER_TABS.PDF">{{ t("project.productTabs.tabs.pdf") }}</TabsTrigger -->
+      <TabsTrigger :value="MULMO_VIEWER_TABS.PDF">{{ t("project.productTabs.tabs.pdf") }}</TabsTrigger >
```

ESLint

[error] Delete `·`

(prettier/prettier)